src/sagas/publicChannels/publicChannels.slice.ts

```diff
@@ -9,8 +9,7 @@ export class PublicChannelsState {
   public channels: EntityState<IChannelInfo> =
     publicChannelsAdapter.getInitialState();
 
-  public currentChannel: string =
-    'zs10zkaj29rcev9qd5xeuzck4ly5q64kzf6m6h9nfajwcvm8m2vnjmvtqgr0mzfjywswwkwke68t00';
+  public currentChannel: string = 'general';
```

Comment:
I would have thought of providing `'general'` as configurable variable(e.g. env) instead of hardcoding it